modules/virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume_{context}"]
+
+= Reclaiming persistent volumes for binding to a new persistent volume
+
+Use this procedure to:
+
+* Release the persistent volume from the persistent volume claim.
+* Erase all data in the share.
+* Create a new persistent volume with a different name,
+maintaining the persistent volume's original connection settings.
+
+.Prerequisite
+* Note the connection details of the persistent volume you want to reclaim.
+
+.Procedure
+. Ensure that the Storage Class *reclaimPolicy* is set to *delete*.
+. Verify that no other resources are using the persistent volume claim, then
+delete the persistent volume claim to release the persistent volume.
++
+----
+$ oc delete pvc/<volume claim>
```

Comment:
s/volume claim/volume_claim

Please show a syntax example.

Reply:
Added underscore. This is the syntax according to dev and QE.